Ensure compatibility with Python 3.8

Original issue opened by @hdiogenes at https://github.com/kytos/kytos/issues/1149.

There 6 tests failing in 3.8 today (and 1980 warnings).

```
======================================================= short test summary info =======================================================
FAILED tests/unit/test_core/test_api_server.py::TestAPIServer::test_shutdown_api - RuntimeError: Working outside of request context.
FAILED tests/unit/test_core/test_api_server.py::TestAPIServer::test_shutdown_api__error - RuntimeError: Working outside of request c...
FAILED tests/unit/test_core/test_atcp_server.py::TestKytosServerProtocol::test_data_received - TypeError: a coroutine was expected, ...
FAILED tests/unit/test_core/test_atcp_server.py::TestKytosServerProtocol::test_connection_lost - TypeError: a coroutine was expected...
FAILED tests/unit/test_core/test_controller.py::TestController::test_start_controller - AssertionError: Calls not found.
FAILED tests/unit/test_core/test_logs.py::TestLogManager::test_custom_formatter - TypeError: expected string or bytes-like object
======================================= 6 failed, 311 passed, 1980 warnings in 92.11s (0:01:32) =======================================
```

Related: #1148

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,7 @@
 language: python
 python:
   - "3.6"
+  - "3.7"
   - "3.8"
 install:
   - pip install pip==20.2.2

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,7 @@
 language: python
 python:
   - "3.6"
+  - "3.8"
 install:
   - pip install pip==20.2.2
   - pip install --use-feature=2020-resolver -r requirements/dev.txt

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ python:
   - "3.7"
   - "3.8"
 install:
-  - pip install pip==20.2.2
+  - pip install --upgrade pip
   - pip install --use-feature=2020-resolver -r requirements/dev.txt
 script:
   - python setup.py lint

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,7 @@ python:
   - "3.8"
 install:
   - pip install --upgrade pip
-  - pip install --use-feature=2020-resolver -r requirements/dev.txt
+  - pip install -r requirements/dev.txt
 script:
   - python setup.py lint
   - python setup.py coverage

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36,py37,py38
+envlist = py36,py37,py38,py39
 
 [testenv]
 whitelist_externals=

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-#envlist = py36,py38
+envlist = py36,py37,py38
 
 [testenv]
 whitelist_externals=

